src/test/statistics/chunk_statistics/range_filter_test.cpp

```diff
@@ -207,6 +222,26 @@ TYPED_TEST(RangeFilterTest, CanPruneOnBounds) {
   EXPECT_TRUE(filter->can_prune(PredicateCondition::GreaterThan, {this->_after_range}));
 }
 
+// Test larger value ranges.
+TYPED_TEST(RangeFilterTest, Between) {
+  const auto filter = RangeFilter<TypeParam>::build_filter(this->_values);
+
+  // This one has bounds in gaps, but cannot prune.
+  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, {this->_max_value - 1}, {this->_after_range}));
```

Comment:
You tested that `BETWEEN this->_max_value - 1 AND this->_after_range` can not be pruned - good! Why not do due diligence and test  `BETWEEN this->_max_value AND this->_after_range` and  `BETWEEN this->_max_value + 1 AND this->_after_range` as well? Why not do the same for the lower end of the value range? And for one or two gaps?

Reply:
Having those tests would scream "I am properly tested for edge cases!!!" much more than a billion randomly generated values :P :P

Reply:
I already have better tests as part of #1321, but I don't know when I will finish the PR. I'll copy your comment to the issue.